skills_DS/api/views.py

```diff
@@ -18,11 +21,22 @@ class FileUploadView(APIView):
 	parser_classes = (FileUploadParser,)
 
 	def post(self, request, format=None):
```

Comment:
Maybe try using `@login_required(login_url='url_here')` instead

https://docs.djangoproject.com/en/4.0/topics/auth/default/#the-login-required-decorator

Reply:
This is related to the login check I mentioned below